src/lib/merkle_ledger/test/test.ml

```diff
@@ -0,0 +1,14 @@
+(* Testing
+   -------
+
+   Component: Merkle ledger tests
+   Subject: Run all Merkle ledger tests
+   Invocation: dune exec src/lib/merkle_ledger/test/test.exe
+*)
+
+let () =
```

Comment:
I would avoid having a single executable. It is better to have multiple executables that can be executed in parallel.
An example can be found inhttps://github.com/MinaProtocol/mina/pull/17334.